userprofile/views.py

```diff
@@ -128,5 +128,47 @@ def get_common_objects(self):
         self.privacy_text = privacy_text
         self.note("privacy_text")
 
+
+class AccessibilityStatementView(UserProfileView):
+    access_mode = ACCESS.ANONYMOUS
+    template_name="userprofile/accessibility.html"
+
+    def get_common_objects(self):
+        super().get_common_objects()
+        lang = "_" + get_language().lower()
+        key = make_template_fragment_key('accessibility_statement', [lang])
+        accessibility_statement = cache.get(key)
+        # TODO: refactor to a helper function
+        if not accessibility_statement:
+            local_template_name = "institution_accessibility_text{}.html"
+            local_template = try_get_template(local_template_name.format(lang))
+            if not local_template and len(lang) > 3:
+                local_template = try_get_template(local_template_name.format(lang[:3]))
+            if not local_template:
+                logger.warning("No localized accessibility statement for language %s", lang)
+                local_template = try_get_template(local_template_name.format(''))
+            if not local_template:
+                logger.error("No local accessibility content at all!")
+
+            local_accessibility_statement = local_template.render() if local_template else _("No local accessibility statement. Please notify the site's owner!")
+
+            system_template_name = "accessibility_issues{}.html"
+            system_template = try_get_template(system_template_name.format(lang))
+            if not system_template and len(lang) > 3:
+                system_template = try_get_template(system_template_name.format(lang[:3]))
+            if not system_template:
+                logger.warning("No localized system accessibility content for language %s", lang)
+                system_template = try_get_template(system_template_name.format(''))
+            if not system_template:
+                logger.error("No system accessibility content at all!")
+
+            system_accessibility_statement = system_template.render() if system_template else _("No system-wide accessibility statement found.")
+
+            accessibility_statement = f'{local_accessibility_statement}{system_accessibility_statement}'
```

Comment:
Is this the new Python string formatting syntax that is only available from Python 3.6 onwards? Our production server is still on Python 3.5 and we must therefore remain compatible with Python 3.5.

Reply:
Ok. 

I assumed the docket container was using the same version as production. 

I switched to this because concatenation of the two strings with `+` fails if the file is missing. 

@Mankro What's the right way to do this in our version of Python?

Reply:
I'm no Python expert, but this seems to work: 

```
str(local_accessibility_statement) + str(system_accessibility_statement
```

Reply:
`_` refers to the function `ugettext_lazy`, but we don't need lazy translation in this case. The template `render` method does not return lazy strings, either. I think you can call `gettext` instead of `_` here and then the string concatenate `+` operator will work directly. You need to import `gettext` to use it. Note that Django has the `format_lazy` function for doing string formatting on lazy strings.

Reply:
I've done this now.